io.github.hakandundar34coding.system-monitoring-center.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.github.hakandundar34coding.system-monitoring-center
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: system-monitoring-center
+
+finish-args:
+  # For X11 shared memory access (higher performance for X11).
+  - --share=ipc
+  # For Wayland access.
+  - --socket=wayland
+  # For X11 fallback.
+  - --socket=fallback-x11
+  # For monitoring data download/upload speed on network cards.
+  - --device=all
+  # For monitoring data download/upload speed on network cards.
+  - --share=network
+  # For host OS commands access (such as 'systemctl' for service details, 'ls /proc' for process list, 'ps' for process information, etc.).
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
You need to add your ID to https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json for this to pass CI.

Reply:
The application ID is added.